src/content.js

```diff
@@ -174,11 +188,78 @@ function apply(options, insertionPoint) {
 	}
 }
 
+function readAllChunks(readableStream) {
+	const reader = readableStream.getReader();
+	const chunks = [];
+
+	return pump();
+
+	function pump() {
+		return reader.read().then(({value, done}) => {
+			if (done) {
+				return chunks;
+			}
+
+			chunks.push(value);
+			return pump();
+		});
+	}
+}
+
+function requestPage(cookies, number) {
+	return fetch(`https://github.com/dashboard/index/${number}?utf8=%E2%9C%93`, {
+		credentials: 'include',
+		Cookie: cookies,
+		headers: {
+			'X-Requested-With': 'XMLHttpRequest'
+		}
+	})
+		.then(data => {
+			const streem = data.body;
+
+			return readAllChunks(streem)
+				.then(data => data
+					.map(chunk => new TextDecoder('utf-8').decode(chunk))
+					.join('')
+				);
+		});
```

Comment:
You don't need all of this manual stream reading, there's a method to get the text:

```js
fetch(...).then(data => data.text())
```